src/Libraries/DSCPython/DSCPython.csproj

```diff
@@ -47,6 +47,14 @@
     <Reference Include="Python.Runtime, Version=2.4.1.0, Culture=neutral, processorArchitecture=MSIL">
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\extern\Python\Python.Runtime.dll</HintPath>
+    </Reference>
+	<Reference Include="Microsoft.Dynamic, Version=1.2.2.0, Culture=neutral, PublicKeyToken=7f709c5b713576e1, processorArchitecture=MSIL">
```

Comment:
this seems wrong - why does DSCPython now require types from the IronPython DLR project?